src/index.tsx

```diff
@@ -6,14 +7,132 @@ import { PersistGate } from "redux-persist/integration/react";
 
 import App from "./App";
 import { store } from "./state";
+import { theme } from "./theme";
+import { Translation } from "react-i18next";
 
 const root = document.getElementById("app");
 const persistor = persistStore(store);
 
+export default class ErrorBoundary extends React.Component<
```

Comment:
Das vielleicht extrahieren in in components?

Reply:
wäre eine Idee, was anderes was mir noch eingefallen ist, sollen wir embed und index wieder in eine Datei zusammenfassen und anhand von Variablen-Konfiguration generieren, ich hatte sie mal kurz verglichen und nicht viele Unterschiede festgestellt. Jedes mal den Code zu duplizieren ist nicht gerade die beste Lösung. 

Müssen es eigentlich `ejs` templates sein? Ich hatte mal geguckt ob wir das Logo in die `index.html` einbetten können, aber das funktioniert mit ejs nicht.

Reply:
In EJS ist aber auch einfach nur HTML mit Platzhaltern, sollte machbar sein. Wir brauchen ein templating, da wir natuerlich Variablen rein schreiben beim build. 

`Embed` und `Index` wuerde ich strikt getrennt halten, sonst haben wir das verwurstet mit laute ifs. So ist es explizit. DIY muss man auch nicht uebertreiben.

Reply:
genau templates auf jedenfall, es gibt halt auch noch underscore und andere templatesprachen die es einbetten können. 

Naja, der unterschied zwischen emded und index ist jetzt nicht sonderlich groß, das eine heißt anders und hat halt keine navigationsleiste.

Reply:
ich mal einen PR dazu auf